pennylane/measurements/classical_shadow.py

```diff
@@ -274,15 +318,69 @@ def shape(self, device=None):
         # and the second indicate the indices of the unitaries used
         return (1, 2, device.shots, len(self.wires))
 
+    def __copy__(self):
+        return self.__class__(
+            self.return_type,
+            obs=copy.copy(self.obs),
+            seed=self.seed,
+            wires=self._wires,
+            eigvals=self._eigvals,
+        )
+
+
+class _ShadowExpval(CustomMeasurement):
+    """Measures the expectation value of an operator using the classical shadow measurement process.
+
+    This has the same arguments as the base class MeasurementProcess, plus other additional
+    arguments specific to the classical shadow protocol.
+
+    Args:
+        args (tuple[Any]): Positional arguments passed to :class:`~.pennylane.measurements.MeasurementProcess`
+        seed (Union[int, None]): The seed used to generate the random measurements
+        H (:class:`~.pennylane.Hamiltonian` or :class:`~.pennylane.operation.Tensor`): Observable
+            to compute the expectation value over.
+        k (int): Number of equal parts to split the shadow's measurements to compute the median of means.
+            ``k=1`` corresponds to simply taking the mean over all measurements.
+        kwargs (dict[Any, Any]): Additional keyword arguments passed to :class:`~.pennylane.measurements.MeasurementProcess`
+    """
+
+    def __init__(self, *args, H, seed=None, k=1, **kwargs):
+        self.seed = seed
+        self.H = H
+        self.k = k
+        super().__init__(*args, **kwargs)
+
+    def process(self, tape, device):
+        """Compute expectation values using classical shadows in a differentiable manner.
+
+        Please refer to :func:`~.pennylane.shadow_expval` for detailed documentation.
+
+        Args:
+            tape (QuantumScript): the tape to be processed
+            device (Device): the device used to process the tape
+
+        Returns:
+            float: expectation value estimate.
+        """
+        bits, recipes = qml.classical_shadow(wires=self.wires, seed_recipes=self.seed).process(
```

Comment:
The `seed_recipes` argument is just a boolean that tells `qml.classical_shadow` whether to generate a random seed to use for random recipe generation (this makes sure that the same recipes are used when the tape is copied, but the samples can still vary). At the moment you can't actually pass in a manual seed yourself. I think there are two options here:

1. Pass `seed_recipes=self.seed is not None` to `qml.classical_shadow`.
2. Allow `seed_recipes` to take integer values in addition to booleans to let the user manually specify a seed.

Reply:
Changed `seed_recipes` to `seed`. Now it is an integer instead of a boolean.